src/Pelagos/Bundle/AppBundle/Resources/views/DataDiscovery/template-dataset-row.html.twig

```diff
@@ -0,0 +1,25 @@
+<table hidden>
```

Comment:
Performance statistics using Firefox's bandwidth simulator, time in seconds. 1st column original, 2nd column this PR
:
Regular 3G
100 20
100 20
100 18

Good-3G
50  10
50  11
50  11

Regular 4G/LTE
21  4
20  5
20  5

DSL
38  8
37  8
38  8

WiFi
7   3
7   3
7   3

Unthrottled
6   3
6   3
6   3

~